src/loaders/trackedEntityLoader.js

```diff
@@ -13,24 +13,13 @@ import { apiFetch } from '../util/api.js'
 import { getDataWithRelationships } from '../util/teiRelationshipsParser.js'
 import { formatStartEndDate, getDateArray } from '../util/time.js'
 
-const fields = [
-    'trackedEntityInstance~rename(id)',
-    'featureType',
-    'coordinates',
-]
-
-// Mapping netween DHIS2 types and GeoJSON types
-const geometryTypesMap = {
-    POINT: 'Point',
-    POLYGON: 'Polygon',
-    MULTI_POLYGON: 'MultiPolygon',
-}
+const fields = ['trackedEntity~rename(id)', 'geometry']
 
 // Valid geometry types for TEIs
-const geometryTypes = Object.keys(geometryTypesMap)
+const geometryTypes = ['Point', 'Polygon', 'MultiPolygon']
```

Comment:
Can you use the constants from util/geojson.js? ('MultiPolygon' constant needs to be created)

Reply:
Addressed here: https://github.com/dhis2/maps-app/pull/3346/commits/96e80ecd45a3438e5c0ae7aa8ee0f9825286d696